cmake/SearchForStuff.cmake

```diff
@@ -213,7 +216,13 @@ if (PKG_CONFIG_FOUND)
   
   if (USE_EXTERNAL_TINYXML)
     message (STATUS "Using system tinyxml.")
-    pkg_check_modules(tinyxml tinyxml)
+    find_package(tinyxml CONFIG)
+    if (tinyxml_FOUND)
+      message (STATUS "tinyxml found via find_package")
+    endif()
+    if (NOT tinyxml_FOUND)
+      pkg_check_modules(tinyxml tinyxml)
+    endif()
```

Comment:
For backward compatibility, can we search via `find_package(tinyxml CONFIG)`  only if `    pkg_check_modules(tinyxml tinyxml)` fails?

Reply:
I will try.